.changeset/selfish-hounds-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/selectable-search-input': minor
+---
+
+We're introducing a new property (XXX) so consumers can customize the different parts of the select element in this component.
```

Comment:
In my example I used (XXX) because we hadn't decided the name of the prop yet, but now that we agreed on naming it `selectCustomComponents` we should update this message.

Reply:
updated here https://github.com/commercetools/ui-kit/pull/2831/commits/b02d49cdbbd9006db106879f0c0bbc0b191790b1 🤦🏾‍♂️

Thank you